src/d3d9/d3d9_device.cpp

```diff
@@ -4280,7 +4280,7 @@ namespace dxvk {
     // The docs say DISCARD and NOOVERWRITE are ignored if the buffer is not DYNAMIC
```

Comment:
Should probably change the previous comment line (not in the patch context) accordingly too

Reply:
Not really. We never respected the docs here and what the docs say hasn't changed.

Reply:
Sorry, I think I was ambiguous. What I was referring to was line 4279, where there is a description of what's going to happen practically.

Reply:
Ah, I see what you mean. You're right, Im gonna change that.